cyctl/internal/describe/modules.go

```diff
@@ -0,0 +1,87 @@
+package describe
+
+import (
+	"fmt"
+
+	"github.com/cyclops-ui/cyclops/cyclops-ctrl/api/v1alpha1/client"
+	"github.com/cyclops-ui/cycops-cyctl/internal/kubeconfig"
+	"github.com/cyclops-ui/cycops-cyctl/utility"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	describeModuleExample = ` 
+	# Describe a single module
+	cyctl describe modules module1
+ 
+	# Describe multiple modules
+	cyctl describe modules module1 module2 module3`
+)
+
+// describeModules describe specified module from the Cyclops API.
+func describeModules(clientset *client.CyclopsV1Alpha1Client, moduleNames []string) {
+	if len(moduleNames) == 0 {
+		modules, err := clientset.Modules("cyclops").List(v1.ListOptions{})
+		if err != nil {
+			fmt.Printf("Error fetching modules: %v\n", err)
+			return
+		}
+		for _, module := range modules {
+			moduleNames = append(moduleNames, module.Name)
+		}
+	}
+
+	for _, moduleName := range moduleNames {
+		module, err := clientset.Modules("cyclops").Get(moduleName)
+		if err != nil {
+			fmt.Printf("Error from server (NotFound): %v\n", err)
```

Comment:
Can you return after this statement so you don't have the rest of the logic in `else`?

Reply:
it's done